src/utils/response.js

```diff
@@ -0,0 +1,54 @@
+const codes = {
+  200: 'sucess',
+  201: 'created',
+  400: 'bad_request',
+  401: 'unauthorized',
+  403: 'forbidden',
+  404: 'not_found',
+  500: 'internal_server_error',
+}
+
+const messages = {
+  200: 'Sucess',
+  201: 'Created',
+  400: 'Bad Request',
+  401: 'Unauthorized',
+  403: 'Forbidden',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
  403: "Forbidden",
```